projects/stylebook-toolkit/src/lib/code-preview-menu/code-preview-menu.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { CodePreviewService } from '../services/code-preview.service';
+import { CodePreviewMenuItem } from '../models/code-preview';
+
+@Component({
+    selector: 'st-code-preview-menu',
+    templateUrl: './code-preview-menu.component.html',
+    host: {
+        '[class]': 'classNames'
```

Comment:
It's ok as it is now, but for a future we may use @Attribute instead of host, because angular is deprecating that.

`@Attribute('class') public class: string = this.classNames;`